langlib/lang.string/src/main/ballerina/string.bal

```diff
@@ -200,6 +282,10 @@ public isolated function equalsIgnoreCaseAscii(string str1, string str2) returns
 #
 # The ASCII white space characters are 0x9...0xD, 0x20.
 #
+# ```ballerina
+# " BALLERINA  ".trim() ⇒ BALLERINA
```

Comment:
I think the output doesn't convey that trailing space was removed. Because of cases like these, should we consider enclosing the output within ""

Reply:
hmm, that's a valid point.

Reply:
Will change this.

Reply:
BTW, there are a few cases like this which I have to show that spaces are available at the end or start of the string. 
There are two options to show this.
1. Include within double quotes
2. Append with another string and show the output.

What is your preference? Thoughts?

Reply:
Since other output doesn't have ""; shall we then append a string? maybe "" start and end? wdyt?

Reply:
hmm, I don't have a strong opinion though. But slightly prefer option #1 :). @MaryamZi your thoughts?

Reply:
It is not only about the `trim` function. We need to consider some other string functions as well, such as `padZero`, `padStart` and `padEnd` cases as well.  Do you guys think appending looks nice for all of them?

Reply:
But in those examples, unless we pad with whitespace, it is comparatively clear, right? IMO, those are OK as is.

Reply:
OK, fine then.

Reply:
Thanks, will update accordingly.